### ISSUE TITLE
Google Maps draws blank on iOS

### ISSUE THREAD

**Issue by alappe** [ionic-team/capacitor-plugins#1022]
### Plugin(s)
@capacitor/google-maps

### Capacitor Version

```
💊   Capacitor Doctor  💊

Latest Dependencies:

  @capacitor/cli: 3.5.1
  @capacitor/core: 3.5.1
  @capacitor/android: 3.5.1
  @capacitor/ios: 3.5.1

Installed Dependencies:

  @capacitor/android: not installed
  @capacitor/core: 3.5.1
  @capacitor/cli: 3.5.1
  @capacitor/ios: 3.5.1

[success] iOS looking great! 👌
```

### Platform(s)
iOS

### Current Behavior
The rendered map stays completely blank on iOS, works in web.

### Expected Behavior
I'd expect to see a map on iOS.

### Code Reproduction
https://github.com/alappe/capacitor_map_example should show the example. One needs to add an API key with iOS Maps SDK enabled~~, mine is missing the credit card for now~~.

### Other Technical Details
Tested on a real device, not simulator.

xCode console:
<img width="430" alt="Screenshot 2022-06-02 at 18 05 13" src="https://user-images.githubusercontent.com/1357848/171676132-1a692306-a1da-4dcd-8ec0-479fd804153c.png">

**Comment by co-dax** [comment]
@alappe I am having the same issue. Are you getting this while using simulator or physical device? It might also be due to the M1-based machines as state at the following link [https://capacitorjs.com/docs/apis/google-maps#ios](url). I testing this in simulator running on Mac OS in VirtualBox and Mac OS is not giving me info about processor type most probably bacause of VirtualBox.

**Comment by alappe** [comment]
I read about the simulator not working on M1, but I also have this issue on the actual device.

Does it work for you on the phone itself?

**Comment by co-dax** [comment]
I did not tested on the phone itself. Let's see if someone else reports anything...

**Comment by jcesarmobile** [comment]
This was fixed in version 4.1.0 https://github.com/ionic-team/capacitor-plugins/pull/1109
